dgraph/cmd/live/run.go

```diff
@@ -703,8 +771,20 @@ func run() error {
 	l := setup(bmOpts, dg, Live.Conf)
 	defer l.zeroconn.Close()
 
+	if err := l.populateNamespaces(ctx, dg, galaxyOp); err != nil {
+		fmt.Printf("Error while populating namespaces %s\n", err)
+		return err
+	}
+
+	if opt.namespaceToLoad != math.MaxUint64 {
```

Comment:
Please create a constant for math.MaxUint64 and give it a better name :)

Reply:
Made `preserveNs` a part of `opt`.